docs/signals/concepts/index.md

```diff
@@ -4,80 +4,224 @@ sidebar_position: 1
+Imagined like this:
 
-User devices and your own systems can request interventions for a list of specific entities, which are then delivered in real-time as they are published.
-You can publish interventions manually using the API, or define them to trigger automatically when [attributes](#attributes) get updated and meet specific criteria.
-There are built-in operations that interventions can perform or that can be handled by Signals SDKs, but the contents of an intervention can contain custom data to use however you need to, and can include current attribute values for dynamic, personalized, actions using the latest real-time data.
+```python
+user.num_page_views_in_last_7_days
+user.last_product_viewed
+user.previous_purchases
+```
 
-For example, a user can subscribe to interventions for their own `domain_userid`, the current `app_id`, the current `page`, and the current `product`, and any interventions published targeting any of those entities get received.
-This enables both individual-level and broadcast-level messaging, so you can offer a specific user a personalized message, while also notifying all users on a specific product page that limited stock is selling fast.
+A single attribute definition could also be associated with multiple entities. For example, an attribute that counts page views might be relevant for user, page, and product entities. Attributes for different entities are distinct, even if they use the same name and definition.
 
-<!-- TODO: Read more about interventions -->
+Imagined like this:
+
+```python
+user.num_page_views_in_last_7_days
+page.num_page_views_in_last_7_days
+product.num_page_views_in_last_7_days
+```
+
+## Attributes can be grouped for ease of management
+
+### Views group attributes
+
+Configure attributes by grouping them into views. Each view is associated with a specific entity, source, version, and owner. It also has other optional metadata.
+
+Choose the source which fits your use case. For example, the attribute `last_product_viewed` is best calculated from events in-stream during a session, while `num_views_in_last_7_days` is best calculated from historical events.
+
+An example configuration for a view based on a user entity:
+
+```mermaid
+flowchart TD
+    User[User: `user_id`] --> Stream
+
+    subgraph Stream[Stream attributes]
+        SA1[number_of_pageviews]
+        SA2[last_product_viewed]
+    end
+
+    Stream --> View[View: `user_attributes_realtime`]
+```
+
+This view could be imagined like this as a table once the attributes are calculated:
+
+| `user_id` | `number_of_pageviews` | `last_product_viewed` |
+| --------- | --------------------- | --------------------- |
+| `abc123`  | 5                     | `"Red Shoes"`         |
+| `def456`  | 10                    | `"Blue Hat"`          |
+
+You can use views individually in your application to retrieve attributes, or combine them into services.
+
+### Services group views
```

Comment:
This is a short term limitation that services can only group views, they should be able to pick and choose individual attributes from different views (e.g., `num_page_views` from `view1` and `total_value` from `view2`).

I think the page `signals/configuration/views-services/` gives a better distinction between views and services – defining vs consuming attributes. Maybe we could somehow combine the explanation here and there?